test/mix/tasks/ecto.gen.migration_test.exs

```diff
@@ -39,6 +39,11 @@ defmodule Mix.Tasks.Ecto.Gen.MigrationTest do
     assert String.match? name, ~r/^\d{14}_my_migration\.exs$/
   end
 
+  test "raises when existing migration exists" do
+    run ["-r", to_string(Repo), "my_migration"]
+    assert_raise Mix.Error, fn -> run ["-r", to_string(Repo), "my_migration"] end
```

Comment:
Let's check the error message also: `assert_raise Mix.Error, ~r"migration can't be created"`.

Reply:
👍  Makes sense to me.  Done, squshed, and pushed.